Uplift/Controllers/HomeViewController+Extensions.swift

```diff
@@ -129,14 +137,22 @@ extension HomeViewController: TodaysClassesListCellDelegate {
 
 }
 
-extension HomeViewController: ProsListCellDelegate {
-
-    func prosListCellShouldOpenProBio(_ proBio: ProBio) {
-        let proDetailViewController = ProBioViewController(pro: proBio)
-        navigationController?.hidesBottomBarWhenPushed = true
-        navigationController?.pushViewController(proDetailViewController, animated: true)
+extension HomeViewController: LookingForListCellDelegate {
+    
+    func lookingForCellShouldTagSearch(at tag: Tag, indexPath: IndexPath) {
+        let cal = Calendar.current
+        let currDate = Date()
+        guard let startDate = cal.date(bySettingHour: 0, minute: 0, second: 0, of: currDate), let classNavigationController = tabBarController?.viewControllers?[1] as? UINavigationController, let classListViewController = classNavigationController.viewControllers[0] as? ClassListViewController else { return }
```

Comment:
can we split the let statements on separate lines
```
guard let startDate = cal.date(bySettingHour: 0, minute: 0, second: 0, of: currDate), 
let classNavigationController...
```

Reply:
See https://github.com/cuappdev/uplift-ios/pull/157#discussion_r324835507